api-examples/lm-text.py

```diff
@@ -14,7 +15,7 @@
     with open(args.text, 'r') as f:
         for line in f:
             text = line.strip().split()
-            texts += [text]
+            texts += text  # consider the whole file as a long sequence input
```

Comment:
I don't think that this is what we want to do. It seems like each line would be a sentence in the batch that we would predict the next token of?

Reply:
In that way, I would add `batch_input` in the `LanguageModelService`, right?

Reply:
I changed it to a batch-input way.